### PR TITLE
add basic flow setup

### PR DESCRIPTION
**Pull request by Licini**
```python
@Node(viewer)
def load_bunny() -> Mesh:
    return Mesh.from_ply(compas.get('bunny.ply'))


@Node(viewer)
def move_bunny(mesh: Mesh) -> Mesh:
    T = Translation.from_vector([10, 0, 0])
    R = Rotation.from_axis_and_angle([1, 0, 0], math.radians(90))
    S = Scale.from_factors([100, 100, 100])
    return mesh.transformed(T * R * S)
```
A basic setup to use decorator to for registering nodes. Now the viewer already automatically updates while adding/deleting nodes or connection changes. Will work on the second part where we can predefine a whole graph.

image

### CODE REVIEW

File: scripts/v120_workflow.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    T = Translation.from_vector([10, 0, 0])
+    R = Rotation.from_axis_and_angle([1, 0, 0], math.radians(90))
+    S = Scale.from_factors([100, 100, 100])
+    return mesh.transformed(T * R * S)
+
+
+# # Define the flow graph (Not working yet)
+# bunny = load_bunny()
+# moved_buuny = move_bunny(bunny)
+# save_bunny(moved_buuny)
```
Comment (brgcode) [inline scripts/v120_workflow.py:28]: would it not be easier to use an actual graph for this?
the edges of the graph could be executed lazily once the viewer is launched?

File: scripts/v120_workflow.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import compas
+import math
+from compas_view2.flow import Node
+from compas_view2.app import App
+from compas.datastructures import Mesh
+from compas.geometry import Scale, Rotation, Translation
+
+viewer = App(viewmode="shaded", show_flow=True, width=800, height=500)
```
Comment (brgcode) [inline scripts/v120_workflow.py:8]: would be good if showing the flow can alternatively also be triggered from the toolbar, for example...

**Review by brgcode — COMMENTED** [review]
cool!

File: scripts/v120_workflow.py (hunk 2 above)
Comment (Licini) [reply scripts/v120_workflow.py:8]: sure of course! will add that option

File: scripts/v120_workflow.py (hunk 1 above)
Comment (Licini) [reply scripts/v120_workflow.py:28]: mmm right, that might be even better, let me try to write up something

**Comment by Licini** [comment]
@brgcode I'd say the basic setup are done. The `Flow` class is now inherited from `Graph`

**Review by brgcode — APPROVED** [review]
LGTM

**Pull request merged**